confluent_kafka/error.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
```

Comment:
why executable?

Reply:
you and your text editors aren't smart enough to work it out from the .py extension?